armory/data/utils.py

```diff
@@ -81,6 +83,20 @@ def curl(url: str, dirpath: str, filename: str) -> None:
         raise subprocess.CalledProcessError
 
 
+def download_requests(url: str, dirpath: str, filename: str):
+    filepath = os.path.join(dirpath, filename)
+    chunk_size = 1024
```

Comment:
This seems like a small chunk_size to me. Should we use a 4k (4096) chunk, as that is a common file block size?